docs/src/interfacer.md

```diff
@@ -1,55 +1,166 @@
+is used for checkpointing the simulation.
+
+### ComponentModelSimulation - optional functions
+- `update_sim!(::ComponentModelSimulation, csf, turbulent_fluxes)`: A
+function to update each of the fields of the component model simulation
+that are updated by the coupler. ClimaCoupler.jl provides defaults of
+this function for both `AtmosModelSimulation` and
+`SurfaceModelSimulation` that update each of the fields expected by
+the coupler. This function can optionally be extended to include
+additional field updates as desired.
+
+### AtmosModelSimulation - required functions
+In addition to the functions required for a general
+`ComponentModelSimulation`, an `AtmosModelSimulation` requires the
+following functions to retrieve and update its fields.
+- `get_field(::AtmosModelSimulation. ::Val{property})`: This getter
+function returns the value of the field property for the simulation
+in its current state. For an `AtmosModelSimulation`, it must be extended
+for the following properties:
+  - `air_density`: air density at the surface of the atmosphere
+  - `air_temperature`: air temperature at the surface of the atmosphere
+  - `energy`: TODO
+  - `F_radiative_TOA`: incoming radiative flux at the top of the atmosphere
+  - `height_int`: TODO
+  - `liquid_precipitation`: liquid precipitation at the surface
+  - `radiative_energy_flux`: TODO
+  - `snow_precipitation`: snow precipitation at the surface
+  - `turbulent_energy_flux`: TODO
+  - `turbulent_moisture_flux`: TODO
+  - `thermo_state_int`: TODO
+  - `uv_int`: TODO
+  - `water`: TODO
+- `update_field!(::AtmosModelSimulation. ::Val{property}, field)`:
+A function to update the value of property in the component model
+simulation, using the values in `field`. This update should
+be done in place. If this function isn't extended for a property,
+that property will remain constant throughout the simulation.
+This function is expected to be extended for the
+following properties:
+  - `co2`: global mean co2
+  - `surface_albedo`: bulk surface albedo over the whole surface space
+  - `turbulent_fluxes`: turbulent fluxes (note: only required to use
+  partitioned turbulent fluxes option - see our `FluxCalculator` module docs for more information)
+
+### SurfaceModelSimulation - required functions
+Analogously to the `AtmosModelSimulation`, a `SurfaceModelSimulation`
+requires additional functions to those required for a general `ComponentModelSimulation`.
+- `get_field(::SurfaceModelSimulation. ::Val{property})`: This getter
+function returns the value of the field property for the simulation at
+the current time. For a `SurfaceModelSimulation`, it must be extended
+for the following properties:
+  - `area_fraction`: the fraction of the simulation grid surface area this model covers
+  - `beta`: TODO
+  - `roughness_buoyancy`: TODO
+  - `roughness_momentum`: TODO
+  - `surface_albedo`: bulk surface albedo
+  - `surface_humidity`: surface humidity
+  - `surface_temperature`: surface temperature
```

Comment:
missing `air_density` and `beta`

Reply:
beta is there a few lines above :) I added air_density too